src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -507,7 +507,17 @@ def generate_config(
 
         alertmanager_user, alertmanager_password = self.mgr._get_alertmanager_credentials()
         prometheus_user, prometheus_password = self.mgr._get_prometheus_credentials()
+        federate_path = self.get_target_cluster_federate_path(targets)
+        cluster_credentials: Dict[str, Any] = {}
+        cluster_credentials_files: Dict[str, Any] = {'files': {}}
         FSID = self.mgr._cluster_fsid
+        if security_enabled and len(targets) > 0:
```

Comment:
`len(targets) > 0 ` check is redundant, in Python you can check directly if a list has elements by `if my_list:`

Reply:
Updated, Thanks @rkachach